ci/build-any-arch-docker-image-task.yml

```diff
@@ -0,0 +1,34 @@
+# task that builds docker images with an option to specify a target platform.
```

Comment:
I would rename the task to `container-image-build-task.yml`

And thanks for the comments, this will make it easier to understand the task in a few months.

Reply:
renamed